### PR TITLE
Add `Select chunks in radius` button, with configuration menu

### PR DESCRIPTION
**Pull request by NotStirred**
image

Adds the ability to select a circle of chunks, within a radius. It's added to the right click menu on the map view
image

Additionally there's a config dialog available in the right click menu.
image
I can't figure out how to get the button to be square though, so it's a little chunkier than the rest of the icons. If any javafx wizard wants to fix that it'd be much appreciated

### CODE REVIEW

**Comment by leMaik** [comment]
I like @Peregrine05's suggestion. @NotStirred What do you think, do you want to change it?

**Comment by jackjt8** [comment]
Just thinking - Given that right click may not be the most accurate, especially when fully zoomed out, could the dialog also have X,Z coordinates so that the user can double check and adjust the centre point?

**Comment by leMaik** [comment]
@NotStirred Would you like to include the two suggestions above into this PR or should we leave that to a future PR? :pray:

**Comment by NotStirred** [comment]
Sure, will implement that now

**Comment by NotStirred** [comment]
New dialog with suggestions:
image

**Comment by JustinTimeCuber** [comment]
Maybe this should have an option to only select chunks within a certain angle? I could imagine loading a bunch of chunks in non-visible directions would use a lot of memory unnecessarily.

**Comment by NotStirred** [comment]
That's what `Select camera visible chunks` is for, I think that does its job well enough

**Comment by JustinTimeCuber** [comment]
That doesn't let you select chunks at a certain circular distance, just a rough "rectangular" distance. I'd find it useful to be able to make a circular selection at a specific distance (aka what this PR does) but without loading non-visible chunks for memory reasons.

**Comment by leMaik** [comment]
@JustinTimeCuber Feel free to create a new issue or submit a PR for this idea. :+1:

File: chunky/src/java/se/llbit/chunky/ui/dialogs/SelectChunksInRadiusDialog.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import se.llbit.chunky.ui.IntegerTextField;
+
+public class SelectChunksInRadiusDialog extends Dialog<ButtonType> {
+  protected float chunkSelectionRadius = 10;
+
+  protected IntegerTextField selectionX;
+  protected IntegerTextField selectionZ;
+
+  public SelectChunksInRadiusDialog() {
+    this.setTitle("Adjust chunk selection radius");
```
Comment (leMaik) [inline chunky/src/java/se/llbit/chunky/ui/dialogs/SelectChunksInRadiusDialog.java:19]:
```suggestion
    this.setTitle("Select chunks in radius");
```

**Review by leMaik — APPROVED** [review]

**Comment by JustinTimeCuber** [comment]
> @JustinTimeCuber Feel free to create a new issue or submit a PR for this idea. +1

Will look into it probably this weekend

**Pull request merged**